### ISSUE TITLE
some transactions showing as conflicted

### ISSUE THREAD

**Issue by underdarkskies** [RavenProject/Ravencoin#177]
After attempting to send multiple units of a single asset to different addresses, I recieved transaction conflicted notifications in the qt gui. 
![image](https://user-images.githubusercontent.com/35243703/43543866-47c8883e-959f-11e8-8613-a1ee91aa8475.png)

After runnning `gettransaction` from the console on an affected transaction i recieved
```

￼
{
  "amount": 0.00000000,
  "fee": -0.00005251,
  "confirmations": -28,
  "trusted": false,
  "txid": "f34a0c87d006b603479cf0601ed426c27fcc8ebe319db0914cf53b070b879e08",
  "walletconflicts": [
    "1fd429d577645dfc68b2f6300e377b68275970d4e44305885ee629515f3d9f1f",
    "644162d4d929b6fa6862a487748ebacaaafe04cf3cfa50d02f965f8860e46938",
    "a53a514993bbd61a6dab166baa645094486eb520a3bf3a67778838dc96ea4072",
    "4c4e15325e31d964be7c95fa0628918c601d89a8e0ddf19222b9478b2624ce7f",
    "9235b386eb3aea648fc8aefa41f4c08d2bb67122bab2d33839bc8a6ff5a744ae"
  ],
  "time": 1533148924,
  "timereceived": 1533148924,
  "bip125-replaceable": "unknown",
  "details": [
  ],
  "asset_details": [
    {
      "asset_type": "transfer_asset",
      "asset_name": "UNDER",
      "amount": 5.00000000,
      "destination": "mm2tgEKBgrewVeBuE9krtTKhWyXLyp2HwY",
      "vout": 0,
      "category": "send"
    }
  ],
  "hex": "0200000002f1b856cc81b2acc07c49425d3fa193ff404aeb9b49ec0c7669226c6df3fe3fbd000000006b483045022100f920be2e6fc893f19a8c1f66fb418f6a2677a914b72f37f994f29b2e2a140b8102206547fb97a920dbb79d8100301f76e788cd06182d8710d75bba9757727211aa6d0121024579f14d0d8f92c9e3fa9411f3be71ec63db0373f6c90e15b629f58c2be91fbcfeffffff56d27dff7f445aed61e7d3399c02ea5946ef01492be963c2df8cd59af00b4f75030000006a4730440220341a849aec62ea8926d3a7ce65c797a20bb96fa3e36d5e2997969ce9922cb5b6022048f8342261c859e4fc3d77da1abeb7333cb97b9d88cf30eed1d0415609dc62280121020954d2637a262cca474f381a896c10bae3615861fc471aff56d47ddc7adadf3dfeffffff0300000000000000002e76a9143c82168155a37bcd4cc6071d6c492601f711881888acc01272766e7405554e4445520065cd1d00000000757d73526a740000001976a914cae65de3dda0fee8ad747fcd46007e9caeb184ad88ac00000000000000002e76a914cae65de3dda0fee8ad747fcd46007e9caeb184ad88acc01272766e7405554e444552000f6e860b0000007524330500"
}
```

**Comment by underdarkskies** [comment]
so it looks like for the raven fee, the same UTXO was used to build the transactions
```
{
  "txid": "9235b386eb3aea648fc8aefa41f4c08d2bb67122bab2d33839bc8a6ff5a744ae",
  "hash": "9235b386eb3aea648fc8aefa41f4c08d2bb67122bab2d33839bc8a6ff5a744ae",
  "version": 2,
  "size": 450,
  "vsize": 450,
  "locktime": 340772,
  "vin": [
    {
      "txid": "754f0bf09ad58cdfc263e92b4901ef4659ea029c39d3e761ed5a447fff7dd256",
      "vout": 0,
      "scriptSig": {
        "asm": "3045022100f259b1fa6ed503dc84a993353cbe4ab0fef80859ac698eeff10fa94218ad3fe4022048d89f94bd1090fb5d3d934e461ace177678c74d29c54d8bd71879460b6c5e0c[ALL] 0232cef3b65ac9bec17609fe13fd84cb848d69b20932eb9328547c83ddd76cc4cb",
        "hex": "483045022100f259b1fa6ed503dc84a993353cbe4ab0fef80859ac698eeff10fa94218ad3fe4022048d89f94bd1090fb5d3d934e461ace177678c74d29c54d8bd71879460b6c5e0c01210232cef3b65ac9bec17609fe13fd84cb848d69b20932eb9328547c83ddd76cc4cb"
      },
      "sequence": 4294967294
    },
    {
      "txid": "754f0bf09ad58cdfc263e92b4901ef4659ea029c39d3e761ed5a447fff7dd256",
      "vout": 3,
      "scriptSig": {
        "asm": "3045022100c47a0ddcabe05ee7f1d25e5238b60f59b1142ac744a66c7205065ade6ffe6d5002200a3ea71fe04451c52d576a03cdb32bfc7d85ad25ad3d787f9300f316316f9db5[ALL] 020954d2637a262cca474f381a896c10bae3615861fc471aff56d47ddc7adadf3d",
        "hex": "483045022100c47a0ddcabe05ee7f1d25e5238b60f59b1142ac744a66c7205065ade6ffe6d5002200a3ea71fe04451c52d576a03cdb32bfc7d85ad25ad3d787f9300f316316f9db50121020954d2637a262cca474f381a896c10bae3615861fc471aff56d47ddc7adadf3d"
      },
      "sequence": 4294967294
    }
  ],
  "vout": [
    {
      "value": 0.00000000,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 01a6ea8cc369ebdfc225cf55358c41e2298e2835 OP_EQUALVERIFY OP_CHECKSIG OP_RVN_ASSET 72766e7405554e4445520065cd1d00000000 OP_DROP",
        "hex": "76a91401a6ea8cc369ebdfc225cf55358c41e2298e283588acc01272766e7405554e4445520065cd1d0000000075",
        "reqSigs": 1,
        "type": "transfer_asset",
        "addresses": [
          "mffgvndHwHsjGDReAawf4WWRjVbgv9dZbR"
        ]
      }
    },
    {
      "value": 0.00000000,
      "n": 1,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 b88e49e078b7a999d9a5918247d7b64fd735ce96 OP_EQUALVERIFY OP_CHECKSIG OP_RVN_ASSET 72766e7405554e444552000f6e860b000000 OP_DROP",
        "hex": "76a914b88e49e078b7a999d9a5918247d7b64fd735ce9688acc01272766e7405554e444552000f6e860b00000075",
        "reqSigs": 1,
        "type": "transfer_asset",
        "addresses": [
          "mxLo5ja33rpHxVQaCANC4vAnWy4j1BN8qf"
        ]
      }
    },
    {
      "value": 2284.99979049,
      "n": 2,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 b88e49e078b7a999d9a5918247d7b64fd735ce96 OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a914b88e49e078b7a999d9a5918247d7b64fd735ce9688ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "mxLo5ja33rpHxVQaCANC4vAnWy4j1BN8qf"
        ]
      }
    }
  ]
}
```

**Comment by underdarkskies** [comment]
not sure if this is intended, maybe i chalk it up to issuing to many transactions, but still its wierd

**Comment by underdarkskies** [comment]
yea so this is reproduceable, if you fire off multiple sends before one of them is confirmed, then you will get conflicted transactions in your waller

**Comment by TronBlack** [comment]
Thank you.   We'll get this fixed.

**Comment by blondfrogs** [comment]
@underdarkskies This should be fixed now with #237
